scripts/select-random-player/select_random_player.lua

```diff
@@ -0,0 +1,30 @@
+local BE_SPECIAL = "Traitor"
+
+function selectRandomPlayer(be_special)
+
+    local players = dm.global_vars:get_var("GLOB"):get_var("alive_player_list")
+    local playerTable = {}
+
+    for _, player in players do
+        local client = player:get_var("client")
+        if not client then continue end
```

Comment:
Nit: selene is going to lint this (you should install the selene extension if you haven't already):

```suggestion
        if not client then
            continue
        end

```

Reply:
I have selene installed but it didn't lint for this, unless you mean it will in the future in which case, fair enough.